components/brave_wallet/browser/zcash/zcash_keyring.cc

```diff
@@ -82,4 +69,17 @@ std::unique_ptr<HDKeyBase> ZCashKeyring::DeriveKey(
   return key->DeriveNormalChild(key_id.index);
 }
 
+absl::optional<std::vector<uint8_t>> ZCashKeyring::SignMessage(
+    const mojom::ZCashKeyId& key_id,
```

Comment:
just want to double check `ZCashKeyring` is under the hood of `keyring_service` which is the service that is accessible for iOS front-end right?

Reply:
Yes, but i believe you don't need to access it directly. Front-end implementation should be very similar to other coins, just using some methods from ZCashWalletService.